pbin.sh

```diff
@@ -100,13 +106,13 @@ set_defaults() {
 	unset reply
 
 	# default to user@hostname
-	name=${SUDO_USER:-$USER}@${HOSTNAME:-$(hostname)}
+	name=${SUDO_USER:-$USER}
```

Comment:
I would say this is fine as is.  If the user doesn't want username@hostname, they can just specify name.